OpenMEEGMaths/include/MatlabIO.H

```diff
@@ -215,7 +215,7 @@ namespace OpenMEEG {
                     os.close();
                     std::remove(name().c_str());
                 }
-                mat_t* mat = Mat_CreateVer(name().c_str(),NULL,MAT_FT_MAT73);
+                mat_t* mat = Mat_CreateVer(name().c_str(),NULL,MAT_FT_DEFAULT);
```

Comment:
@massich this is what I changed to have the tests pass on windows.

Reply:
from `matio_pubconf.h`:

Reply:
it is fine to use `MAT_FT_DEFAULT`, instead of adding a parameter to the             `void write(std::ofstream& os, const LinOp& linop) const` to allow writing in different matlab formats in runtime.

Reply:
like this the writing format is not hardcoded and can be changed in compilation time.